src/components/assets/styles/account.scss

```diff
@@ -100,8 +100,8 @@
   display: flex;
   justify-content: flex-end;
   column-gap: 24px;
-  @media (min-width: $xl) {
-    column-gap: 80px;
+  @media (max-width: $sm) {
```

Comment:
Please fix all the 'max-width' to 'min-width' as we are using it on this repo.

Reply:
@impelcrypto fixed it ([15bec29](https://github.com/AstarNetwork/astar-apps/pull/935/commits/15bec290973a72bd4d5b9f22347180a99d9c7bbb))